docs/tutorials/jokes.md

```diff
@@ -1705,9 +1706,9 @@ And here's what we have with that now:
 
 I want to call out something specific in my solution. Here's my loader:
 
-```tsx lines=[8-10]
+```tsx lines=[2,8-10]
 type LoaderData = {
-  jokeListItems: Array<{ id: string; name: string }>;
+  jokeListItems: Array<Pick<Joke,"id" | "name">>;
```

Comment:
Although this is correct, it may be confusing for a new comer to typescript in this on boarding tutorial.